src/plugins/julia/templates/lr.template.jl

```diff
@@ -0,0 +1,243 @@
+module SyntaxParser
+
+#=
+  LR parser generated by the Syntax tool.
+
+  https://www.npmjs.com/package/syntax-cli
+
+    npm install -g syntax-cli
+
+    syntax-cli --help
+
+  To regenerate run:
+
+    syntax-cli \
+      --grammar ~/path-to-grammar-file \
+      --mode <parsing-mode> \
+      --output ~/ParserClassName.jl
+=#
+
+# --------------------------------------------------------------
+# Exports: The parse method, if concerned about disambiguating with other
+# parse methods in your use case, use 'import' instead of 'using'
+export parse
+
+# --------------------------------------------------------------
+# Shared includes
+using DataStructures
+
+# Constants and globals
+const EOF = "\$"
+yytext = ""
+yylength = 0
+__res = nothing
+__loc = nothing
+should_capture_locations = {{{CAPTURE_LOCATIONS}}}
+
+# Types
+struct SyntaxError <: Exception
+    msg::AbstractString
+end
+
+function Base.showerror(io::IO, err::SyntaxError)
+    print(io, err.msg)
+end
+
+Base.@kwdef mutable struct yyLoc
+    startoffset
+    endoffset
+    startline
+    endline
+    startcolumn
+    endcolumn
+end
+
+Base.@kwdef mutable struct StackEntry
+    symbol
+    semanticvalue
+    loc
+end
+
+# --------------------------------------------------------------
+# Module includes provided by the grammar.
+{{{MODULE_INCLUDE}}}
+
+# --------------------------------------------------------------
+# Tokenizer.
+
+{{{TOKENIZER}}}
+
+#=
+  Throws default "Unexpected token" exception, showing the actual
+  line from the source, pointing with the ^ marker to the bad token.
+  In addition, shows line:column location.
+=#
+function throwunexpectedtoken(tokenizerData::TokenizerData, symbol, line::Integer, column::Integer)
+    throw(SyntaxError(string("Incorrect Syntax\n\n", split(tokenizerData.initstring, "\n")[line], "\n", " "^(column - 1), "^\nUnexpected Token: \"", symbol, "\" at ", line, ":", column, ".")))
```

Comment:
We can keep the `"Incorrect Syntax"` (and in general custom error messages per plugin) if you'd like for Julia's output, however we should probably standardize the output to be the consistent with other Syntax plugins.

Reply:
Removed the "Incorrect Syntax", I think this is aligned now with how the cs plugin reports errors.